articles/tableau-vs-power-bi/index.md

```diff
@@ -0,0 +1,57 @@
+Business intelligence software is a top priority item in the toolkit of almost every business team. This is due to the significance of the tool in data collection, storage, and visualization. Analysis from BI tools is used to make strategic business decisions. But before running a project, you have to ensure that you use the right business intelligence product. This article intends to compare the two most commonly used data visualizations tools: Tableau and Power BI.
+
+### An Overview of Tableau
+
+[Tableau](https://www.tableau.com/) is an analytics tool that helps to combine and integrate data from many sources into visualizations. Users can access this data in a single desktop environment with the use of Tableau Desktop. Alternatively, the data can be accessed through a shared dashboard. Tableau&#39;s user-friendly interface allows users to analyze data easily, create visualizations and reports, and share insights.
+
+Basically, Tableau Desktop acts as a report development tool. You can use this tool to create reports and format your data into visualizations. You can then organize visualizations into a dashboard for personal use or share the same through Tableau Server. The Tableau Server portal is secure, and you can share your dashboard with others from any mobile device or browser.
+
+### An Overview of Power BI
```

Comment:
Add a section indicating the various parameters this article will be comparing the software on. That will give a clean and polished introduction, to begin with.